app/src/main/java/movierating/App.java

```diff
@@ -34,7 +39,7 @@
         return testCasesList;
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws MovieReleasedException {
```

Comment:
`MovieReleasedException` is a checked exception. You need not pass it more up because no one more can handle it. I think you can handle in this main.